acceptance/acceptance_test.go

```diff
@@ -455,18 +462,22 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	printedRepls := false
 
 	// Compare expected outputs
-	for relPath := range outputs {
+	for _, relPath := range outputs {
 		doComparison(t, repls, dir, tmpDir, relPath, &printedRepls)
 	}
 
 	// Make sure there are not unaccounted for new files
 	files := ListDir(t, tmpDir)
```

Comment:
Here's a way to implement this without making abstract Replacement type:

- Walk over all output files, extract all UUIDs
- Prepend replacements for those UUIDs to config.Repls.
- Proceed normally

The only thing that won't see this replacement is diff.py (but we can fix that too, see my other comment).